Vagrantfile

```diff
@@ -24,7 +25,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     puppet.options        = '--verbose --debug'
     puppet.facter         = {
       'mysql_root_password'  => 'vagrant',
-      'wwwroot'              => '/www'
+      'wwwroot'              => DEST_FOLDER,
+      'db_username'          => 'username',
+      'db_password'          => 'password',
+      'db_name'              => 'wp_devnet',
+      'wp_url'               => PROJECT_NAME,
```

Comment:
The WP URL should be `"#{config.vm.hostname}.local"` and the title could be `PROJECT_NAME`.